src/com/sun/ts/tests/jms/ee20/resourcedefs/annotations/files/ServletClient.java.src

```diff
@@ -27,7 +27,7 @@ import jakarta.servlet.annotation.WebServlet;
 import java.util.Enumeration;
 import java.util.Properties;
 import java.util.ArrayList;
-import javax.transaction.*;
+import jakarta.transaction.*;
```

Comment:
I could be overlooking something but I don't see any javax.transaction or jakarta.transaction classes referenced in ServletClient.java.src.